website/docs/howto.md

```diff
@@ -0,0 +1,278 @@
+---
+id: howto
+title: How to use Gatekeeper
+---
+
+Gatekeeper uses the [OPA Constraint Framework](https://github.com/open-policy-agent/frameworks/tree/master/constraint) to describe and enforce policy. Look there for more detailed information on their semantics and advanced usage.
```

Comment:
This may be fine just to get the PR out, but I think we should clean up the documentation split a bit.

The constraint framework should definitely have the most detailed info on how things work, but much of it will likely only be relevant for people interested in writing targets or enforcement points.

Maybe this means we create a section of the CF documentation explicitly on using constraints/templates? Maybe we add documentation on how to write templates to the library?

I'm not sure what the best solution is. Whatever we do, we should make sure user's aren't overwhelmed and we should avoid repeating ourselves (which the linking does nicely).

Reply:
> we create a section of the CF documentation explicitly on using constraints/templates?

I think we should add this doc in the CF repo.